### PR TITLE
[MIG][13.0]sales_report_product_image:Migrated module from 12.0 to 13.0.

### PR DESCRIPTION
**Pull request by ChavdaRiddhi**

### CODE REVIEW

File: sales_report_product_image/tests/test_sales_report_product_image.py [hunk 1]
```diff
@@ -22,5 +22,5 @@ def test_sale_report_product(self):
                 'pricelist_id': self.env.ref('product.list0').id,
                 'print_image': 'True',
                 'image_sizes': 'image_medium',
-                'image_small': self.product.image_small,
+                'image_small': self.name.image_1920,
```
Comment (JayVora-SerpentCS) [inline sales_report_product_image/tests/test_sales_report_product_image.py]: any reason for 1920?

Comment (ChavdaRiddhi) [reply sales_report_product_image/tests/test_sales_report_product_image.py]: @JayVora-SerpentCS In version 13.0  **hr**  module employee image field name is **image_1920**,so we took that field.

Comment (JayVora-SerpentCS) [reply sales_report_product_image/tests/test_sales_report_product_image.py]: This is not HR. 1920 is size taken dynamicallly. Better you check with Chiragbhai.

Comment (ChavdaRiddhi) [reply sales_report_product_image/tests/test_sales_report_product_image.py]: @JayVora-SerpentCS I see , it's my mistake. i will correct it.

Comment (ChavdaRiddhi) [reply sales_report_product_image/tests/test_sales_report_product_image.py]: Improved Code as per the suggestion.

Comment (JayVora-SerpentCS) [reply sales_report_product_image/tests/test_sales_report_product_image.py]: @YogendrasinhChavda web_tree_image ni kharekhar jarur 6?

Comment (YogendrasinhChavda) [reply sales_report_product_image/tests/test_sales_report_product_image.py]: @JayVora-SerpentCS: Actually that OCA module ( web_tree_image ) is to show the images in List view, but now we managed by adding the Height for the image field in the tree view. so, that module is not needed.
We Removed that from dependency.

**Pull request merged**